Neos.Neos/Classes/Cache/ContentCacheFlusherInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Cache;
+
+use Neos\ContentRepository\Core\ContentRepository;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
+use Neos\ContentRepository\Core\SharedModel\Node\NodeAggregateId;
+use Neos\Media\Domain\Model\AssetInterface;
+
+/**
+ * The interface for services flushing content caches triggered by node changes.
+ *
+ * It is called when the projection changes: In this case, it is triggered by
+ * {@see GraphProjectorCatchUpHookForCacheFlushing} which calls this method..
+ *   This is the relevant case if publishing a workspace
+ *   - where we f.e. need to flush the cache for Live.
+ */
```

Comment:
```suggestion
 * @internal this extension point is experimental 
 */
```

what do you say? is this interface subject to change or can we safely declare it as `@api`

Reply:
internal sounds good for now